chore(docs): mailchimp - how to get campaign id (#49)
watson-developer-cloud/assistant-toolkit: integrations/extensions/starter-kits/mailchimp/readme.md

Note that `campaignid` is a session variable that must match the campaign you have created. If you have created your campaign via API, then in the response you should have an `id` property, that should be saved in `campaignid`. If you haven't created the campaign via the API, then you have to use the API to obtain the campaign id. To do this, [list the campaigns](https://mailchimp.com/developer/marketing/api/campaigns/list-campaigns/) then find your campaign.

> Here is an example of a cURL you can run to obtain the list of campaigns. 
> Please replace `server_prefix` as explained above and `auth` should be your private API key

Reviewer: ```suggestion
Here is an example of a cURL you can run to obtain your list of campaigns. 
Please replace `server_prefix` as explained above, and `auth` with your private API key.
```
